spec.html

```diff
@@ -910,6 +910,22 @@ <h1>Invoking Syntax-Directed Operations</h1>
         <p>Regardless of the convention, the invocation causes the execution of the algorithm that the syntax-directed operation associates (either explicitly or implicitly, see <emu-xref href="#sec-implicit-definitions-for-chain-productions"></emu-xref>) with the subject Parse Node's production. Thus, within any algorithm of the syntax-directed operation, the subject Parse Node is necessarily an instance of one of the production(s) associated with that algorithm.</p>
       </emu-clause>
 
+      <emu-clause id="sec-referring-to-parse-nodes">
+        <h1>Referring to Parse Nodes</h1>
+        <p>Although a syntax-directed operation is always invoked with a Parse Node, the heading for a syntax-directed operation does not declare a parameter to receive it. Instead, each definition uses various phrases to refer to the subject Parse Node and its children, depending on the production(s) associated with that definition. The general idea is that the algorithm refers to nonterminals in the associated production(s), and thereby refers to the Parse Nodes that correspond to those nonterminals for a particular invocation.</p>
+        <p>To refer to the subject Parse Node:</p>
+        <ul>
+          <li>If all of the associated productions have the same left-hand side nonterminal, then that nonterminal is used with “this” (e.g., “this |ExportDeclaration|”).</li>
+          <li>Otherwise, “this Parse Node” is used.</li>
+        </ul>
```

Comment:
The way this is written sounds like editorial guidance, which is unnecessary in the spec. The goal here is just to list notation that they may run across. Can you reword to simplify it?

Reply:
It does kind of sound like editorial guidance, but it's really more a description of editorial variations.

I could reword it, but I'm doubtful that would simplify it. I'll think about it.